### PR TITLE
Fix config bug and optimize issue creation

### LINKED ISSUE #21: Error when filtering selected projects
When I attempted to use this on a new machine, after I was prompted to enter my Snyk Organization, I ran into an error:

```
✔ Snyk token · xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
✔ Snyk organization · xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
TypeError: Cannot read property 'filter' of undefined
    at exports.init (/.../snyk-github-issue-creator/cli/config.js:156:38)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async /.../snyk-github-issue-creator/cli/index.js:27:5
```

Since I didn't have a `configstore` file, [the `snykProjects` variable was undefined](https://github.com/elastic/snyk-github-issue-creator/blob/4f78ff4d8cee7615befe700239627283286bddd6/cli/config.js#L102), and Node ran into an error after that when attempting to [filter the projects](https://github.com/elastic/snyk-github-issue-creator/blob/4f78ff4d8cee7615befe700239627283286bddd6/cli/config.js#L155-L158).

### LINKED ISSUE #22: Allow continuing to create issues
Since #18 was merged to use the new Snyk API, the CLI can be very slow to enumerate the list of issues in your project(s).
This can be especially annoying if you want to create multiple GitHub issues, as you have to re-run the CLI every time.

It would be great if, after creating a GitHub issue, the CLI would ask if you want to continue creating issues -- if yes, then it could show the list of issues it already fetched, without calling the Snyk API again.

### PR DESCRIPTION
**Pull request by jportner**
Resolves #21
Resolves #22

* Fixes error related to snykProjects variable when using `snyk-github-issue-creator` on a new machine that doesn't have a local `configstore` already
* Adds prompt to continue creating new issues

Example prompt:

```
✔ Pick a vulnerable package · H|696  - parse-link-header 1.0.1
Found 1 vulnerabilities...
Format: [Severity]|[Priority score] - [Package name] [Package Version] - [Vuln title] - [Vuln ID]

1. H|999 - bar 1.0.1 - Regular Expression Denial of Service (ReDoS) - SNYK-JS-...
 * projectname > foo > bar@1.0.1

✔ Add "1. H|999 - bar 1.0.1 - Regular Expression Denial of Service (ReDoS) - SNYK-JS-..." to batch? (y/N) · true
The following GitHub issues were created:
- "projectname - Regular Expression Denial of Service (ReDoS) in bar 1.0.1" https://github.com/orgname/reponame/issues/1234
? Pick another issue? (y/N) › false
```

By default, the prompt response is "N", but if you pick "Y" it takes you back to the issue list and allows you to pick another.

### CODE REVIEW

**Comment by jportner** [comment]
I signed the CLA just now, if someone could trigger a new check 🙏

File: cli/index.js [hunk 1]
```diff
@@ -171,95 +171,127 @@ async function createIssues() {
...
     issues = Object.values(reduced);
 
-    if (conf.batch) {
-        // filter down to the package that was picked
-        issues = (await getBatchProps(issues)).issues;
-    }
-
-    if (conf.autoGenerate) {
+    let continuePrompting = true;
+    while (continuePrompting) {
```
Comment (jportner) [inline cli/index.js:175]: This diff is terrible, but I just moved this whole block into a `while` loop.

File: cli/index.js [hunk 2]
```diff
@@ -171,95 +171,127 @@ async function createIssues() {
...
 }
 
 function getIssueBody(issue) {
-    const { id, issueData: { url, description }, projects } = issue;
+    const {
+        id,
+        issueData: { url, description },
+        projects,
+    } = issue;
     return `This issue has been created automatically by a source code scanner
```
Comment (jportner) [inline cli/index.js:295]: This just got reformatted by prettier :+1:

**Review by jportner — COMMENTED** [review]
Author's notes for reviewers

**Review by watson — APPROVED** [review]
Hi Joe 👋

Thanks for fixing the `undefined` error! And great idea with the continued prompting 💯

File: cli/index.js (hunk 1 above)
Comment (watson) [reply cli/index.js:175]: I clicked "ignore whitespace" in the diff and it looked a little better

Comment (jportner) [reply cli/index.js:175]: TIL!

**Pull request merged**